### PR TITLE
add expandability to budget rows

### PR DESCRIPTION
**Pull request by brendansudol**
### This pull request changes...
- allows you to toggle visibility of budget row details

Preview:
https://cl.ly/rWUR

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: web/src/components/Budget.js [hunk 1]
```diff
@@ -1,23 +1,74 @@
...
+        )}
+      </Fragment>
+    );
+  }
+}
 
 NumberRow.propTypes = {
-  title: PropTypes.string.isRequired,
-  n: PropTypes.number.isRequired
+  n: PropTypes.number,
```
Comment (mgwalker) [inline web/src/components/Budget.js]: This prop name is tickling my linter senses.  My instinct is it's just `n` as a placeholder and will get a better name once the "real" prop is in place.  Just flagging it for now - no need to change anything.

File: web/src/components/Budget.js [hunk 2]
```diff
@@ -29,7 +80,7 @@ const Filler = () => (
   </Fragment>
 );
 
-const ActivitiesPage = () => (
+const BudgetTable = () => (
```
Comment (mgwalker) [inline web/src/components/Budget.js]: 😆

File: web/src/components/Budget.js (hunk 1 above)
Comment (brendansudol) [reply web/src/components/Budget.js]: couldn't agree more

**Comment by brendansudol** [comment]
thx mgw -- going to walk through with design tomorrow

**Pull request merged**